src/HealthGPS/riskfactor.cpp

```diff
@@ -3,9 +3,8 @@
 namespace hgps {
 
 RiskFactorModule::RiskFactorModule(
-    std::map<RiskFactorModelType, std::unique_ptr<RiskFactorModel>> models,
-    const RiskfactorAdjustmentModel &adjustments)
-    : models_{std::move(models)}, adjustment_{adjustments} {
+    std::map<RiskFactorModelType, std::unique_ptr<RiskFactorModel>> models)
+    : models_{std::move(models)} {
```

Comment:
I guess this map needs to be ordered?

Reply:
Actually no, they're explicitly indexed. I just missed it!

Reply:
Having said that, there's just two elements: static and dynamic. In this case, a plain ol' `std::map` would in fact be faster than a `std::unordered_map`t. In any case, this will eventually be replaced with an ordered container, like `std::vector` for #231.

Reply:
Good point! Forget about it for now then